CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ Ansible Changes By Release
   * dnos10_command
   * dnos10_config
   * dnos10_template
+- dnos6
```

Comment:
I thought since 1 < 6 its in alphabetical order, let me know if you want to change it.